src/App.js

```diff
@@ -1,26 +1,25 @@
 import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
```

Comment:
any specific reason not to use `useSelector/useDispatch` instead of connecting components?

Reply:
We are already using useSelector/useDispatch